das/state.go

```diff
@@ -111,7 +124,11 @@ func (s *coordinatorState) handleResult(res result) {
 		}
 		s.failed[h] = nextRetry
 	}
-	s.checkDone()
+
+	// cleanup retry from processed heights
```

Comment:
```suggestion
	// cleanup processed heights from retry
```

Reply:
It is all way around. We need to clean up global retry map from heights that has been processed by job. I've added small explanation to this comment section